webserver/webserver/src/main/java/io/helidon/webserver/ListenerConfigBlueprint.java

```diff
@@ -327,6 +327,15 @@ interface ListenerConfigBlueprint {
      */
     Optional<Context> listenerContext();
 
+    /**
+     * Enable support for proxy protocol for this socket.
```

Comment:
I think it would be nice to have a bit of an explanation of what is meant by proxy prototocol, if there is a spec, maybe a link to it?

Reply:
Improved